sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -105,6 +106,15 @@ private[hive] class HiveClientImpl(
 
   private class RawHiveTableImpl(override val rawTable: HiveTable) extends RawHiveTable {
     override lazy val toCatalogTable = convertHiveTableToCatalogTable(rawTable)
+
+    override def hiveTableProps(containsStats: Boolean): Map[String, String] = {
```

Comment:
why do we need this parameter?

Reply:
I'm not sure if we need all hive table properties in some other places?